docs/containers/docker-compose.md

```diff
@@ -0,0 +1,217 @@
+
+- On the **Debug** tab, choose the **Configuration** dropdown, choose **New Configuration** and select the `Docker Attach` configuration template for appropriate platform.  For example, **.NET Core Docker Attach (Preview)**.
+- Configure the debugging port in `docker-compose.debug.yml`. This is set when you create the file, so you might not need to change it. In the example below for a Node.js app, port 9229 is used for debugging on both the host and the container.
+
+   ```yml
+    version: '3.4'
+
+    services:
+      node-hello:
+        image: node-hello
+        build: .
+        environment:
+          NODE_ENV: development
+        ports:
+          - 3000
+          - 9229:9229
+        command: node --inspect=0.0.0.0:9229 ./bin/www
+    ```
+
+    Python apps do not include the *docker-compose.debug.yml* file.
+
+- If you have multiple apps, you need to change the port for one of them, so that each app has a unique port. You can point to the right debugging port in the `launch.json`, and save the file. If you omit this, the port will be chosen automatically. 
+
+    ```json
+        "configurations": [
+            {
+                "type": "node",
+                "request": "attach",
+                "name": "Docker: Attach to Node",
+                "remoteRoot": "/usr/src/app",
+                "port": 9229
+            },
+            // ...
+        ]
+    ```
+
+- On the **Debug** tab in the **Configuration** dropdown, select the configuration you just created.
+
+Once `launch.json` is configured, select your new launch configuration as the active configuration. In the **Debug** tab, find the new configuration in the **Configuration** dropdown.  
+
+![Screenshot of Configuration dropdown](./images/compose/docker-compose-configuration.png)
+
+The extension generates the launch configuration in the `launch.json` file, but you need to review it and make any appropriate changes for your scenario. For example, the exposed port typically maps to the same port locally if you're only debugging one app at a time, but if you have multiple apps and multiple containers, you'll need to map the debug ports to different ports on the host.
+
+To start debugging a running app or service, first make sure the **Attach** configuration you created is selected in the **Debug** tab configuration dropdown, then launch the debugger in the usual way. From the **Debug** tab, choose the green arrow (**Start** button) or use `kb(workbench.action.debug.start)`.
+
+![Screenshot of starting debugging](images/compose/docker-compose-attach.png)
+
+When you attach, the web browser doesn't open automatically, so you need to start the browser on the host and navigate to the app at `http://localhost:{port}`, where the `port` is found by inspecting the .json configuration for the running container as described previously.  
```

Comment:
The part about the browser not opening by default is valid, but I would suggest clarifying that it applies only to services that expose an HTTP endpoint that returns HTML, which is not always the case (e.g. it could be a REST API service exposing a gRPC endpoint). 

Also, one can use `docker port` command to find out what host port a given container port was mapped to, for example:
```shell
$ docker port apisvc 5000

127.0.0.1:49155
````